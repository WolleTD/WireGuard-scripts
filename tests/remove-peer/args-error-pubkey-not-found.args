iCJSbobpCRHCG2/WP9D1/viGlv+WrNpWtd1XkRzyrFs=
