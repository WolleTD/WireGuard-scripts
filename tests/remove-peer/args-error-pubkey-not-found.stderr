No peer matching the search string was found!
